charm/ksql/icon.svg

```diff
@@ -0,0 +1,279 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Not a fan of these empty generated icons. Might be nice to replace it with a KSQL SVG file, but otherwise I'd remove it.